addons/message_update_v2.17/enable.js

```diff
@@ -6,6 +6,16 @@
     api.addon.date = (api.settings.updateTime.getTime() / 1000);
   }
 
+
+  // Macos v2.16.0 requires a web-based update.
```

Comment:
Explaining the reason for this in the comment may be useful to the reader.

Reply:
I'm sorry @vinocher! I'm saw this right as I merged. Will send up a follow up.